src/rosdoc_lite/msgenator.py

```diff
@@ -104,7 +104,7 @@ def _generate_raw_text(raw_text):
 
 def _generate_msg_text_from_spec(package, spec, msg_context, buff=None, indent=0):
     if buff is None:
-        buff = cStringIO.StringIO()
+        buff = io.StringIO()
```

Comment:
For full compatibility we have a conditional import to do this: http://wiki.ros.org/python_2_and_3_compatible_code